README.md

```diff
@@ -1 +1,5 @@
-# Review-your-project-from-trials-
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
I think It might be better to replace this README file with one in relation to the project. Here is the link that can help you create a professional README file https://github.com/microverseinc/readme-template

Reply:
Nice comment with a helpful link🔥🔥🔥